account_invoice_user_authorization_payment/views/account_move_views.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<odoo>
+
+    <record id="user_authorization_payment_tree" model="ir.ui.view">
+        <field name="name">account.move.tree (User Authorization Payment)</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_in_invoice_tree"/>
```

Comment:
In this view, but also in others, a group restriction should be added (e.g. "Show Full Accounting Features" group)